client/src/components/settings/sound.tsx

```diff
@@ -28,6 +49,19 @@ export default function SoundSettings({
           toggleSoundMode(sound ? false : true);
         }}
       />
+      <label htmlFor='volumeslider'>
+        {t('settings.sound-volume')} {volumeDisplay}
```

Comment:
```suggestion
        {t('settings.sound-volume')} <span aria-hidden='true'>{volumeDisplay}</span>
```

I agree with Tom, we should fix this.